subsys/mpsl/mpsl_init.c

```diff
@@ -171,6 +172,12 @@ static int mpsl_lib_init(const struct device *dev)
 		return err;
 	}
 
+#if CONFIG_MPSL_FEM_DEVICE_CONFIG_254
+	mpsl_fem_device_config_254_shall_apply_set(true);
+#else
+	mpsl_fem_device_config_254_shall_apply_set(false);
+#endif
+
```

Comment:
This function is mispelled: `mpsl_fem_device_config_254_apply_set` is the correct one.

You could simplify this by doing:

```suggestion
	mpsl_fem_device_config_254_apply_set(
		IS_ENABLED(CONFIG_MPSL_FEM_DEVICE_CONFIG_254));
```

Reply:
Fixed